src/blockchains/eos/eos-watch.test.ts

```diff
@@ -0,0 +1,53 @@
+require("dotenv").config()
+
+import "jest-extended"
+import {
```

Comment:
Maybe we should group these imports. It'll look better than separating them with a newline.